proposals/0173-Read-Generic-Network-Signal-data.md

```diff
@@ -35,12 +45,23 @@ Instead of validating vehicle data items against APIs XML, SDL would rely on a d
 
 * _array_ : To speciify if the vehicle data item/param response is an array.
 * _mandatory_ : To specify if the vehicle data param is mandatory to be included in response for overall vehicle data item.
+* _name_ is required for top level vehicle data items while _dataType_, _reference_ & _mandatory_ are required fields for vehicle data & sub-params. However _array_ can be omitted, If omitted, _array_ defaults to *false*.
+* _params_ is recursive array of sub-params for a vehicle data, see example below for structure definition.
+* _since_, _until_ are String values and are optional per vehicle data item. _removed_, _deprecated_ are Boolean values and are optional per vehicle data item. _minvalue_, _maxvalue_, _minsize_, _maxsize_, _minlength_, _maxlength_ are Integer values and are optional per vehicle data item.
+  * _Custom/OEM Specific_ vehicle data parameters that are not a part of the rpc spec should not have any version related tags included (since, until, removed, deprecated), these vehicle data parameters would not be able to have the same versioning system as the rpc spec since any version number supplied would not be the version associated with any known public rpc spec.
+
+The proposed Policy Table structure assumes that SDL enums are defined in the RPC Spec and are _not_ defined within the Policy Table.
 
-* _id_ is required for top level vehicle data items while _dataType_, _reference_ & _mandatory_ are required fields for vehicle data & sub-params. However _array_ and _params_ can be omitted, If omitted, _array_ defaults to *false*.
 
 #### Schema file location and updates
 A base JSON schema file for SDL core and proxy will be kept locally and an additional one in the OEM cloud. OEMs can update the schema file with OEM specific items, which SDL core would be able to download over the cloud along with policy updates; SDL proxy would still process the local(static) copy of schema for existing data items. For OEM specific additional items, the OEM app would process a generic _Object_ to retrieve vehicle data item response _struct_. This new design would ensure flexibility to add OEM specific vehicle data items while maintaining access control via policies the same way it is done currently.
 
+* OEM's schema file should always be baselined with latest version of standardized schema file from open SDL. OEM can then further add OEM specific custom vehicle data items. This ensures that any version of core which implements this proposal would be able to process the latest version of standardized vehicle data item schema, and hence would be able to support all future additions for new standardized vehicle data items.
```

Comment:
Update to:

OEM's schema file should always be baselined with latest version of standardized schema file from open SDL. **OEMs** can then further add OEM specific custom vehicle data items. This ensures that any version of core **that** implements this proposal would be able to process the latest version of standardized vehicle data item schema, and hence would be able to support all future additions for new standardized vehicle data items.

Reply:
Done